_includes/integrations/react-quick-start.md

```diff
@@ -96,22 +92,102 @@ This procedure requires:
 
       {{site.productname}} can be self-hosted by: deploying {{site.productname}} independent of the React application, or bundling {{site.productname}} with the React application.
 
-      * **Deploy {{site.productname}} independent of the React application**
+      * **Deploy {{site.productname}} independent of the React application using `tinymceScriptSrc`**
+
+        To use an independent deployment of {{site.productname}}, add the property `tinymceScriptSrc` to specify the path to the {{site.productname}} script, such as:
+        ```jsx
+        <Editor tinymceScriptSrc="/path/to/tinymce.min.js" />
+        ```
+
+        To do this with create-react-app put the tinymce distribution in `./public`
+        and reference the path to the `public` folder using the environment 
+        variable `process.env.PUBLIC_URL`, such as:
+         ```jsx
+         <Editor tinymceScriptSrc={process.env.PUBLIC_URL + '/tinymce/tinymce.min.js'}>
+         ```
+
+      * **Deploy {{site.productname}} independent of the React application with a script tag**
 
-        To use an independent deployment of {{site.productname}}, add a script to either the `<head>` or the end of the `<body>` of the HTML file, such as:
+        An alternative method is to add a script to either the `<head>` or the
+        end of the `<body>` of the HTML file, such as:
         ```html
         <script src="/path/to/tinymce.min.js"></script>
         ```
 
-        To use an independent deployment of {{site.productname}} with the create a React application, add the script to `/path/to/tinymce-react-demo/public/index.html`.
+        To do this with create-react-app, add the script tag to `./public/index.html`.
+        
+        Normally the tinymce distribution would be put in the `public` folder
+        and referenced using the URL `%PUBLIC_URL%/tinymce/tinymce.min.js`, such as:
+        ```html
+        <script src="%PUBLIC_URL%/tinymce/tinymce.min.js"></script>
+        ```
 
-        For information on self-hosting {{site.productname}}, see: [Installing {{ site.productname }}]({{site.baseurl}}/general-configuration-guide/advanced-install/).
+        For information on self-hosting {{site.productname}}, see: [Installing {{site.productname}}]({{site.baseurl}}/general-configuration-guide/advanced-install/).
 
       * **Bundling {{site.productname}} with the React application using a module loader**
 
+          Bundling with a module loader can be complex and error prone so it is
```

Comment:
Well... 

The problem comes down to module loaders assuming that everything is on one page...
In TinyMCE we have a separate subpage where we need CSS and Javascript.

Additionally TinyMCE loads a lot of things asynchronously to avoid a large upfront download. This includes things like CSS, plugins and parts of plugins that are lazy loaded (ie codeview).

I do provide an example below the statement which will get them 99% of the way there, however to be honest they're still likely to run into problems, hence I'd rather like to discourage them upfront then have to answer the support case...

Reply:
Also I can't fix it in the react integration unless you want me to provide editors with a preset selection of plugins.

Reply:
Dang. Fair enough. I think my suggestion is strong-enough discouragement.